src/components/ConversationSettings/BotsSettings.vue

```diff
@@ -0,0 +1,164 @@
+<!--
+  - @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @author Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="bots-settings">
+		<p class="bots-settings__hint">
+			{{ t('spreed', 'Bots are available for installation for administrators from Apps tab, and could be enabled by moderators for specific chats') }}
+		</p>
+
+		<ul v-if="bots?.length" class="bots-settings__list">
+			<li v-for="bot in bots"
+				:key="bot.id"
+				class="bots-settings__item">
+				<div class="bots-settings__item-info">
+					<span class="bots-settings__item-name">
+						{{ bot.name }}
+					</span>
+					<span class="bots-settings__item-description">
+						{{ bot.description ?? t('spreed', 'Description is not provided') }}
+					</span>
+				</div>
+				<NcButton class="bots-settings__item-button"
+					:type="bot.state ? 'primary' : 'secondary'"
+					:disabled="isBotForceEnabled(bot)"
+					@click="toggleBotState(bot)">
+					{{ toggleButtonTitle(bot) }}
+				</NcButton>
+			</li>
+		</ul>
+		<p v-else class="bots-settings__hint">
+			{{ t('spreed', 'There are currently no available bots for this conversation') }}
+		</p>
+	</div>
+</template>
+
+<script>
+
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
+
+import { BOT } from '../../constants.js'
+import { useBotsStore } from '../../stores/bots.js'
+
+export default {
+	name: 'BotsSettings',
+
+	components: {
+		NcButton,
+	},
+
+	props: {
+		/**
+		 * The conversation's token
+		 */
+		token: {
+			type: String,
+			required: true,
+		},
+	},
+
+	setup() {
+		const botsStore = useBotsStore()
+
+		return {
+			botsStore,
+		}
+	},
+
+	computed: {
+		bots() {
+			return this.botsStore.getBots(this.token)
+		},
+	},
+
+	created() {
+		this.botsStore.updateBotsInformation(this.token)
+	},
+
+	methods: {
+		isBotForceEnabled(bot) {
+			return bot.state === BOT.STATE.FORCE_ENABLED
+		},
+
+		toggleBotState(bot) {
+			if (this.isBotForceEnabled(bot)) {
+				return
+			}
+
+			this.botsStore.toggleBotState(this.token, bot)
+		},
```

Comment:
What about adding a loading state here, like in Apps page?
![image](https://github.com/nextcloud/spreed/assets/25978914/c24579b6-6952-443f-9ec0-4c1a6da84e50)